app/models/post.rb

```diff
@@ -1,2 +1,12 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
- [ ] Great job creating a method that updates the posts counter for a user. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code might look like this 👇🏻 :


   ```ruby
     # app/models/post.rb
     after_save :update_post_counter
   
     private
   
     def update_post_counter
       author.increment!(:posts_counter)
     end
   ```

- [ ] Note: Make sure you to use the  `author.increment!` and not  `user.increment!`